.ci/check_links.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+"""
+This script is like check-basics.py, but specific to the documentation.
```

Comment:
probably, this comment not related to this script, because we have not check_basic.py :)